modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/context/BaseCommerceContextHttp.java

```diff
@@ -306,7 +306,9 @@ public int getCommerceSiteType() {
 
 	@Override
 	public long getCPConfigurationListId(long groupId) throws PortalException {
-		if (!FeatureFlagManagerUtil.isEnabled("LPD-10889")) {
+		if (!FeatureFlagManagerUtil.isEnabled(
+				_portal.getScopeGroupId(_httpServletRequest), "LPD-10889")) {
+
```

Comment:
we should be grabbing companyId here right?

Reply:
yeah, you're right. I've updated the PR